server/py/services/alerts/tests/unit/crud/test_alerts.py

```diff
@@ -410,10 +410,65 @@ async def test_store_alert_update_time(
             alert_data=alert_data,
         )
 
-        alert = services.alerts.crud.Alerts().get_enriched_alert(
+        alert = services.alerts.crud.Alerts().get_alert(
             session=db,
             project=project,
             name=alert_name,
         )
         assert alert.updated is not None
         assert alert.updated > alert.created.replace(tzinfo=timezone.utc)
+
+    @pytest.mark.asyncio
+    @unittest.mock.patch.object(
+        framework.utils.singletons.db.SQLDB, "update_alert_activation"
+    )
+    @unittest.mock.patch.object(
+        services.alerts.crud.AlertActivation, "store_alert_activation"
+    )
```

Comment:
instead of setting it separately in the test, do:
```suggestion
    @unittest.mock.patch.object(
        framework.utils.singletons.db.SQLDB, "update_alert_activation", return_value=None
    )
    @unittest.mock.patch.object(
        services.alerts.crud.AlertActivation, "store_alert_activation", return_value=None
    )
```